jsonify_stored/models/jsonify_stored_mixin.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class JsonifyStored(models.AbstractModel):
+    """Stores the data to export.
+       When inheriting this class, _export_xmlid should be set with an export xml_id.
+       Note that this xml_id has to exist before the model init,
+       i.e. either be in a dependency module, or created in a pre_init_hook
+       (however it's fine to modify an existing one in the module data).
+    """
+
+    @api.depends(lambda self: self._jsonify_get_export_depends())
```

Comment:
do we really need this? You already write the flag when ir.export gets updated.

Reply:
@simahawk This is required. The depends trigger is computed from the ir.exports definition linked to the model. The flag is set into ir.exports when the definition is updated only.